### PR TITLE
New views in Staking 

### PR DESCRIPTION
**Pull request by kirilantevski**
- new views and polished scripts

### CODE REVIEW

File: contracts/EscrowAndStaking/StakingThales.sol [hunk 1]
```diff
@@ -206,6 +206,43 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
...
+        // SNX staked more than base reward
+        return stakedSNX >= baseReward
+            ? 15
+            : stakedSNX.mul(15).div(baseReward);
+    }
+    
+    function getAMMBonus(address account) public view returns (uint) {
+        uint baseReward = getBaseReward(account);
+        return _getTotalAMMVolume(account) >= baseReward.mul(10)
+            ? 12
```
Comment (grujawork) [inline contracts/EscrowAndStaking/StakingThales.sol:233]: can this be some kind of property or it is fix

File: contracts/EscrowAndStaking/StakingThales.sol [hunk 2]
```diff
@@ -206,6 +206,43 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
...
+            : _getTotalAMMVolume(account).div(AMM_EXTRA_REWARD_PERIODS).mul(12).div(baseReward.mul(10));
+    }
+    
+    function getThalesRoyaleBonus(address account) public view returns (uint) {
+        return (address(thalesRoyale) != address(0) && thalesRoyale.hasParticipatedInCurrentOrLastRoyale(account))
+            ? 3
```
Comment (grujawork) [inline contracts/EscrowAndStaking/StakingThales.sol:239]: same here

**Review by grujawork — APPROVED** [review]
for me this is totally good :)

File: contracts/EscrowAndStaking/StakingThales.sol (hunk 1 above)
Comment (kirilantevski) [reply contracts/EscrowAndStaking/StakingThales.sol:233]: fast code, it's a view. 
The main code is in another function

File: contracts/EscrowAndStaking/StakingThales.sol (hunk 2 above)
Comment (kirilantevski) [reply contracts/EscrowAndStaking/StakingThales.sol:239]: same from up

File: contracts/EscrowAndStaking/StakingThales.sol (hunk 1 above)
Comment (grujawork) [reply contracts/EscrowAndStaking/StakingThales.sol:233]: Cool

**Pull request merged**